stdlib/jdbc/src/main/java/org/ballerinax/jdbc/actions/BatchUpdate.java

```diff
@@ -45,4 +45,8 @@
                 rollbackAllInFailure, strand);
         return (MapValue<String, Object>) batchUpdateStatement.execute();
     }
+
+    private BatchUpdate() {
+
+    }
```

Comment:
Why do we need to add these?

Reply:
Since this is not supposed to be instantiated added private constructor.